src/components/custom-script/custom-script.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="custom-script">
+    <!-- we show a <noscript> element if the user has javascript disabled, this is necessary because some functionality might be broken due to javascript being disabled and its important that users are notified why it isn't working -->
+    <!-- this needs to be v-html instead of your typical {{ someValue }} interpolation because of a bug that content in noscript does not seem to be evaluated by Vue -->
```

Comment:
Not sure this is a bug, feels like intended behaviour? Maybe we can just mention it's not being evaluated?

Reply:
You're right, not a bug!